src/components/Progress/__stories__/Progress.stories.tsx

```diff
@@ -1,17 +1,18 @@
 import React, {useEffect} from 'react';
 import {Meta, Story} from '@storybook/react';
 import {Progress, ProgressProps} from '../Progress';
+import {ProgressTheme} from 'src';
```

Comment:
Why imported from src and not from import above?

Reply:
Because VCS's auto-completion :)

Reply:
Can u fix this?